zypp/misc/YamlTestcaseHelpers.h

```diff
@@ -197,6 +197,18 @@ namespace yamltest::detail {
         }, err );
         if ( !success ) return false;
       }
+      else if ( key == ("vendors") )
+      {
+        bool success = readListInlineOrFromFile( [&target]( const YAML::Node & dataNode, std::string * err ) {
+	  std::vector<std::string> vlist;
+	  for ( const auto & node : dataNode )
+	    vlist.push_back( node.as<std::string>() );
+	  if ( ! vlist.empty() )
+	    target.vendorLists.push_back( std::move(vlist) );
```

Comment:
Why do you first push into a std::vector<std::string> instead of directly pushing into target.vendorLists?

Reply:
Otherwise I had to remove empty vectors again from target.vendorLists (which is vector< vector<string> >)